puppet/zulip/files/nagios_plugins/zulip_app_frontend/check_cron_file

```diff
@@ -9,8 +9,7 @@ import time
 if False:
     from typing import Tuple
 
-def nagios_from_file(results_file):
-    # type: (str) -> Tuple[int, str]
+def nagios_from_file(results_file: str) -> Tuple[int, str]:
```

Comment:
This will blow up right at import time, because the import of `Tuple` is inside an `if False`.

We can probably remove the `if False` check now -- Python 3.4, the earliest we support, has a `typing` module in the stdlib.

The same applies to other files under `puppet/`.